packages/Updater/UpdaterServiceProvider.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+namespace Updater;
+
+use Illuminate\Foundation\Application;
+use Illuminate\Foundation\Support\Providers\EventServiceProvider;
+use Illuminate\Support\Facades\Event;
+use Illuminate\Support\ServiceProvider;
+use Updater\Events\PostUpdateEvent;
+use Updater\Events\PreUpdateEvent;
+use Updater\Listeners\ArtisanPreUpdateListener;
+use Updater\OutputAggregator\InternalOutputRecorder;
+use Updater\OutputAggregator\OutputRecorderInterface;
+use Updater\Services\CommandExecutor\ArtisanExecutor;
+use Updater\Services\CommandExecutor\ArtisanExecutorInterface;
+use Updater\Services\CommandExecutor\ComposerExecutor;
+use Updater\Services\CommandExecutor\ComposerExecutorInterface;
+use Updater\Services\CommandExecutor\ExecutorInterface;
+use Updater\Services\CommandExecutor\RecordedExecutor;
+use Updater\Services\CommandExecutor\ShellExecutor;
+use Updater\Services\DateTimeProvider;
+use Updater\Services\DateTimeProviderInterface;
+use Updater\Services\SourceControl\GitSourceControlService;
+use Updater\Services\Updater\UpdaterService;
+use Updater\Services\Updater\UpdaterServiceInterface;
+use Updater\Listeners\ArtisanPostUpdateListener;
+use Updater\Listeners\ComposerPostUpdateListener;
+
+class UpdaterServiceProvider extends ServiceProvider
+{
+    protected $listen = [
+        PostUpdateEvent::class => [
+            ComposerPostUpdateListener::class,
+            ArtisanPostUpdateListener::class,
+        ],
```

Comment:
Ligt de volgordelijkheid van de listeners hiermee vast of kan deze nog verschillen per fired event?